find common characters 2.py

```diff
@@ -0,0 +1,23 @@
+from collections import Counter
+from typing import List
+
+class Solution:
+    def commonChars(self, words: List[str]) -> List[str]:
+        char_array = [0] * 26
+        offset = ord('a')
+
+        original_letters = Counter(words[0])
+        for char, count in original_letters.items():
+            char_array[ord(char) - offset] = count
+
+        for i in range(1, len(words)):
+            letters = Counter(words[i])
+
+            # Update char_array based on the characters present in each word
```

Comment:
Comments:
The code has some comments, which is good. However, some of the comments could be improved to be more descriptive. For instance, instead of the comment # this block of code helps track the letters in the first word, it would be better to mention that this part updates charArray based on the characters present in the first word.